README.rst

```diff
@@ -49,16 +49,23 @@ The -L flag can be used to whitelist certain words that are comma-separated plac
 
     codespell -S, --skip=
 
-Comma-separated list of files to skip. It accepts globs as well.  Examples:
+The -S (--skip) flag is used to skip over a comma-separated list of files and/or directories. It accepts globs as well. **Important notes:** Do not append a '/' to a directory you want to skip. This renders the entry for that particular directory ineffective. Also 
```

Comment:
It wants to be "to skip, this renders". Assuming there aren't more words to go after the trailing also, then it should be "Important note:" (singular)

Reply:
Addressed in 90cf99f